src/components/ProcessCreate/Steps/Census.tsx

```diff
@@ -1,25 +1,35 @@
-import { Box, Flex, Icon, Tab, TabList, TabPanel, TabPanels, Tabs, Text } from '@chakra-ui/react'
+import { Box, Flex, Icon, Tab, TabList, TabPanel, TabPanels, Tabs, Text, useDisclosure } from '@chakra-ui/react'
+import { useState } from 'react'
 import { useTranslation } from 'react-i18next'
+import { CgMoreO } from 'react-icons/cg'
 import { Check } from '~theme/icons'
 import { CensusType, useCensusTypes } from '../Census/TypeSelector'
+import { UnimplementedCensusType, useUnimplementedCensusTypes } from '../Census/UnimplementedTypeSelector'
+import ModalPro from '../ModalPro'
 import { StepsNavigation } from './Navigation'
 import { StepsFormValues, useProcessCreationSteps } from './use-steps'
 import Wrapper from './Wrapper'
-// import checkIcon from '/assets/check-icon.svg'
 
 export interface CensusValues {
   censusType: CensusType | null
 }
 
 export const Census = () => {
   const { t } = useTranslation()
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
   const { form, setForm } = useProcessCreationSteps()
   const { defined, details } = useCensusTypes()
+  const { defined: definedUnim, details: detailsUnim } = useUnimplementedCensusTypes()
   const { censusType } = form
 
+  const [reason, setReason] = useState('')
+  const [showProCards, setShowProCards] = useState(false)
```

Comment:
No need to change it but as a FYI: for this kind of usage you can use the chakra-ui `useDisclosure` hook (used a few lines above actually). Simply rename the vars used to others and that's it.